spec/integration/workflow-steps.feature

```diff
@@ -0,0 +1,46 @@
+ Feature: Workflow steps
+
+  The selected Step's properties can be viewed & some of them can be edited.
+  The selected Step's Inputs' properties can be viewed & some of them can be edited.
+  Steps of any Workflow are listed, they can be selected, and their name, version, icon & badges are displayed.
+
+  Background:
+    Given editor is open
+
+  Scenario: Step's selected
+    When I click on "First step"
+    Then I should see "GitHub Status" in "Step Title"
+    Then I should see "2.2.2" in "Step Version"
```

Comment:
Is testing an exact version number likely to need updating all of the time? Is this adding anything over testing under this  that the steps inputs have been rendered?

Reply:
this step version is locked in a test fixture file, it has nothing to do with editor version, so this is essentially predefined test data.

Reply:
Ah ok, sorry I'm not familiar with the setup of the workflow editor. So is there a locally running API this is running against then?

Reply:
yep, we already had a binary built with rice (https://github.com/GeertJohan/go.rice) which acts as a server for the tests. https://github.com/bitrise-io/bitrise-workflow-editor/blob/9afb393ec3047716a79780821dbdb9c27029173b/package.json#L15